pkg/scheduler/internal/cache/cache.go

```diff
@@ -39,8 +38,10 @@ var (
 // It automatically starts a go routine that manages expiration of assumed pods.
 // "ttl" is how long the assumed pod will get expired.
 // "stop" is the channel that would close the background goroutine.
-func New(ttl time.Duration, stop <-chan struct{}) Cache {
-	cache := newSchedulerCache(ttl, cleanAssumedPeriod, stop)
+// "newNodeInfo" is a function creating a NodeInfo object
+// "balancedVolumesEnabled" signals if BalanceAttachedNodeVolumes feature is enabled
```

Comment:
This feature is alpha and disabled by default since 1.11. Should we remove it?

@Huang-Wei @ahg-g

Reply:
+1 to remove it. But I'm not quite sure whether and how widely it's used.

@ravisantoshgudimetla any thoughts on removing it?

Reply:
We should remove it indeed. 

@ingvagabund why are we making this change in this PR, this has nothing to do with cutting the dependency on `k8s.io/kubernetes/pkg/apis/core/v1/helper`

Reply:
So the `New` function is not updated twice once the dependency on `k8s.io/kubernetes/pkg/features` is dropped. I put the change into the same commit.

Reply:
I undone the change touching the feature gate to leave it for another PR removing dependency on k8s.io/kubernetes/pkg/features

Reply:
I don't think anyone used it except for Red Hat. Let me check internally if it is still being used. This is a request that came from our online environment folks back in the day. If it is still being used, I'll promote it to beta or else remove it.

Reply:
@ravisantoshgudimetla did you check this? I very much want to remove this piece of code as it makes 2 ongoing refactorings more challenging.

Reply:
I did. I am waiting for a response. Please don't block reviews/merges for this. I or someone from RH can follow-up with a removal PR if needed.

Reply:
Update: Red Hat is not actively using it anymore. So, feel free to remove this plugin. I can get started on removing this plugin next week unless someone wants to pick it up.